packages/scanner/src/page.ts

```diff
@@ -41,6 +41,10 @@ export class Page {
             return { unscannable: true, error: `Cannot scan ${url} because it is not a html page.` };
         }
 
+        if (!response.ok()) {
+            return { error: `Error accessing ${url}: ${response.status()} ${response.statusText()}` };
```

Comment:
Can we change the message similar to:
The URL request returned unsuccessful response ...